vignettes/tips_and_tricks.qmd

```diff
@@ -0,0 +1,56 @@
+---
+title: "Tips & Tricks "
+format:
+  html:
+    css: custom.css
+tbl-colwidths: [20,20,50,10]
+---
+
+This page collects miscellaneous tips and tricks for **tinyplot**.
```

Comment:
Could we add: "Please feel free to suggest or add more tips via our [GitHub repo](https://github.com/grantmcdermott/tinyplot/issues)."